.github/workflows/api-tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   api-tests: 
     timeout-minutes: 20
     runs-on: ubuntu-latest
-    if: github.repository == 'JiscSD/octopus' && ${{ !github.event.pull_request.draft }}
+    if: github.repository != 'Jisc/octopus' && ${{ !github.event.pull_request.draft }}
```

Comment:
the draft logic seem broken now... this is currently a draft PR and the api-tests are running

Reply:
fixed